### DISCUSSION TITLE
"No scope" mod for Catch the Beat

### DISCUSSION THREAD

**Discussion by Moltanicaa** [ppy/osu#15128] [Ideas] [answered]
Why not transfer the new mod to CtB?

**Comment by bdach** [comment]
i assume the idea would be to hide the catcher?

not sure it should be named the same because it's not exactly the same thing, but i guess that is an idea.

**Reply by peppy** [reply]
I'd probably call it something else. Or we'd be renaming the mod itself to be more generic.

**Comment by Semro** [comment, marked as answer]
Implemented via https://github.com/ppy/osu/pull/15540, probably requires renaming as suggested above.
